### PR TITLE
SUM-198 | implement qa fixes

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- SUM-198 | implement qa fixes

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Review code

# Associated Issues and/or People
- SUM-198

### CODE REVIEW

File: src/components/paragraphs/rows/three-column.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const ThreeColumn = ({items}: {items: ParagraphUnion[]}) => {
   }
 
   return (
-    <div className="centered grid gap-10 @12xl:grid-cols-3 md:gap-20" {...draftProps}>
+    <div className="centered grid gap-10 lg:grid-cols-3 lg:gap-20" {...draftProps}>
```
Comment (pookmish) [inline src/components/paragraphs/rows/three-column.tsx:19]: Can we still use container queries? interior pages will have different breakpoints than full width pages.

**Review by pookmish — CHANGES_REQUESTED** [review]
Looks like you have a couple links that aren't correctly positioned
image
image

Also there's not enough color contrast on several pill banners.

File: src/components/paragraphs/rows/three-column.tsx (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/paragraphs/rows/three-column.tsx:19]: Ooo great catch! I'll update it to the container query version 🤔

**Comment by pookmish** [comment]
Still have links in the wrong place and bad contrast on pill banners
https://summer-git-feature-sum-198-qafixes-stanford-summer-session.vercel.app/style-samples
image

File: src/components/paragraphs/sum-course-filter/course-filtering-form.tsx [hunk 2]
```diff
@@ -246,7 +246,7 @@ const RefinementInput = ({
...
+              className="block h-[2.4rem] w-[2.4rem] cursor-pointer rounded-full border-2 border-black-50 outline-none transition-all checked:border-lagunita-light checked:bg-[length:0px_0px] checked:text-lagunita checked:ring-4 checked:ring-inset checked:ring-white hocus:border-lagunita hocus:bg-[#99D7E1] hocus:bg-transparent hocus:outline-none hocus:ring-0 hocus:ring-offset-0 checked:hocus:border-lagunita checked:hocus:bg-lagunita-dark checked:hocus:text-lagunita-dark checked:hocus:ring-4 checked:hocus:ring-inset checked:hocus:ring-[#99D7E1] hocus-visible:outline-none group-hocus:border-lagunita group-hocus:bg-transparent checked:group-hocus:bg-lagunita-dark group-hocus:checked:bg-[#99D7E1] checked:group-hocus:text-lagunita-dark"
```
Comment (pookmish) [inline src/components/paragraphs/sum-course-filter/course-filtering-form.tsx:249]: I don't see what the `group` is for. There's no group parent, and `checked:group-hocus` on the `<input>` doesn't make much sense to me. I don't think any of these changes are working at this time.

**Review by pookmish — CHANGES_REQUESTED** [review]

File: src/components/paragraphs/sum-course-filter/course-filtering-form.tsx (hunk 2 above)
Comment (rebeccahongsf) [reply src/components/paragraphs/sum-course-filter/course-filtering-form.tsx]: I went in to clean up remove the `group-hocus` (along with some `checked:hocus`) that weren't doing anything. I also moved the accessibility fixes from #56 to this PR instead 😃 It's ready for another looksie when you have the chance! Thank you, Mike!

File: src/components/paragraphs/sum-course-filter/course-filtering-form.tsx [hunk 3]
```diff
@@ -244,9 +244,9 @@ const RefinementInput = ({
...
+          <label key={refinementOption.value} className="mb-5 flex cursor-pointer items-center gap-5 last:mb-0">
             <input
-              className="block h-[2.4rem] w-[2.4rem] cursor-pointer rounded-full border-2 border-black-50 outline-none transition-all checked:border-lagunita-light checked:bg-[length:0px_0px] checked:text-lagunita checked:ring-4 checked:ring-inset checked:ring-white checked:hover:border-lagunita focus:border-lagunita focus:bg-transparent focus:outline-none focus:ring-0 focus:ring-offset-0 checked:focus:border-lagunita checked:focus:bg-lagunita-dark checked:focus:text-lagunita-dark checked:focus:ring-4 checked:focus:ring-inset checked:focus:ring-[#99D7E1] focus-visible:outline-none group-hover:border-lagunita group-hover:bg-transparent checked:group-hover:bg-lagunita-dark checked:group-hover:text-lagunita-dark group-hover:checked:ring-[#99D7E1]"
+              className="block h-[2.4rem] w-[2.4rem] cursor-pointer border-2 border-black-70 outline-none transition-all checked:border-lagunita-light checked:text-lagunita hocus:border-lagunita hocus:bg-[#99D7E1] hocus:text-lagunita-dark hocus:ring-4 hocus:ring-[#99D7E1]"
```
Comment (pookmish) [inline src/components/paragraphs/sum-course-filter/course-filtering-form.tsx:249]: Oh i like that much better. But the `#99d7e1` doesn't have enough contrast. Can we either underline the label or make it darker during hocus?

**Review by pookmish — CHANGES_REQUESTED** [review]
This is very close, but just needs a little fixup.

**Review by pookmish — APPROVED** [review]

**Pull request merged**